third_party/gpus/cuda_configure.bzl

```diff
@@ -209,6 +213,38 @@ def _enable_cuda(repository_ctx):
   return False
 
 
+def _cuda_nvcc_path(repository_ctx):
+  """Finds the cuda toolkit directory.
+
+  Args:
+    repository_ctx: The repository context.
+
+  Returns:
+    A speculative real path of the cuda toolkit install directory.
+  """
+  cuda_nvcc_path = _DEFAULT_CUDA_NVCC_PATH
+  if _CUDA_NVCC_PATH in repository_ctx.os.environ:
+    cuda_nvcc_path = repository_ctx.os.environ[_CUDA_NVCC_PATH].strip()
+  if not repository_ctx.path(cuda_nvcc_path).exists:
+    auto_configure_fail("Cannot find cuda nvcc path.")
+  return str(repository_ctx.path(cuda_nvcc_path).realpath)
+
+def _cuda_include_folder_path(repository_ctx):
+  """Finds the cuda toolkit directory.
+
+  Args:
+    repository_ctx: The repository context.
+
+  Returns:
+    A speculative real path of the cuda toolkit install directory.
+  """
+  cuda_include_path = _DEFAULT_CUDA_INCLUDE_PATH
```

Comment:
Previously this looks like it was...

cuda_toolkit_path + "/include"

but now ...

_DEFAULT_CUDA_INCLUDE_PATH = "/usr/local/cuda"

Should this be "/usr/local/cuda/include" ?